install/mod_pagespeed_test/image_rewrite_with_flush/image_rewrite_with_flush.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+  <title>Title of the document </title>
+</head>
+<body>
+<ul>
+  <li data-thumb="a.jpg">
```

Comment:
This system test is great; thanks for doing this!

But I'd also consider writing a unit test.  See https://github.com/pagespeed/mod_pagespeed/blob/master/net/instaweb/rewriter/css_filter_test.cc#L2297 for a CSS test with a flush in an inline style element as a model.  

Copy that and modify it according to your needs into https://github.com/pagespeed/mod_pagespeed/blob/master/net/instaweb/rewriter/image_rewrite_filter_test.cc

If you have any questions please don't hesitate to ask.